FrontEnd/components/Sidebar/Left/index.vue

```diff
@@ -121,19 +121,22 @@ import {
   DocumentTextIcon,
   PencilIcon,
   LogoutIcon
-} from "@heroicons/vue/solid"; // for solid icons
+} from "@heroicons/vue/solid"; 
 import { computed } from "vue";
-import { useUserStore } from "~/stores/useUserStore";
+import { useUserStore } from "~/stores/user";
+import { usePostStore} from "~/stores/post";
+
 
 const userStore = useUserStore();
+const postStore = usePostStore()
 const colorMode = useColorMode();
 
-const userId = computed(() => {
-  return userStore.userInfo.id;
-});
-const user = computed(() => {
-  return userStore.userInfo.name;
-});
+const userId = userStore.userId
+const user =  userStore.name
+
```

Comment:
recheck lines 134 and 135. As per the Pinia conception, it is recommended to use [storeToRefs](https://pinia.vuejs.org/core-concepts/#Destructuring-from-a-Store)

Reply:
used storeToRefs(store) as mentioned in the update to extract properties from the store while keeping its reactivity while not calling any actions. please review the latest commit for the changes